### ISSUE TITLE
Implement Registry

### ISSUE THREAD

**Issue by EXtremeExploit** [Vita3K/Vita3K#1965]
Currently there is no registry, the registry stores some system configuration like screen brightness, psp emulation and other stuff, there are some stubs for some cases but they are not good enough, an example is `sceRegMgrGetKeyInt`

```cpp
EXPORT(int, sceRegMgrGetKeyInt, const char *category, const char *name, int *buf) {
    if ((std::string(category) == "/CONFIG/PSPEMU") && (std::string(name) == "emu_list_flag")) {
        STUBBED("Stubbed");
        *buf = 1;

        return 0;
    } else
        return UNIMPLEMENTED();
}
```

Its just a simple if statement for a single case, however, what if you wanted to add more cases? what if its `/CONFIG/PSPEMU/` instead of `/CONFIG/PSPEMU` (trailing / at the end)? Both are valid and behave the same. What about reading an int as a string?. We could store these outside of the emulated path, something like (in linux) `.local/share/Vita3K/registry/`. same goes for windows, however im not sure if the registry is case sensitive or not, if it is case sensitive then windows would need a layer to check for case sensitivity on files/folders.

**Comment by scribam** [comment]
@EXtremeExploit I think registry has been implemented, starting with #2722

**Comment by Zangetsu38** [comment]
Oh no see it
